### PR TITLE
[HB-6922] BidMachine SDK 2.4.1

### PR DESCRIPTION
**Pull request by cb-jpadilla**
- Updated BidMachine SDK to 2.4.1.
- Updated Changelog, README, build script to 4.2.4.1.0

### CODE REVIEW

**Review by chauduyphanvu — COMMENTED** [review]
Does this have the `canShow()` usage?

**Comment by cb-jpadilla** [comment]
@chauduyphanvu it currently does. Will help with the certification. Although, I will need to rebase this PR with `main` once PR https://github.com/ChartBoost/chartboost-mediation-android-adapter-bidmachine/pull/4 has been merged and released. 👍🏼

File: BidMachineAdapter/build.gradle.kts [hunk 1]
```diff
@@ -73,7 +73,7 @@ dependencies {
     "remoteImplementation"("com.chartboost:chartboost-mediation-sdk:4.0.0")
 
     // Partner SDK
-    implementation("io.bidmachine:ads:2.4.0")
+    implementation("io.bidmachine:ads:2.4.1")
```
Comment (chauduyphanvu) [inline BidMachineAdapter/build.gradle.kts:76]: Don't forget to also change the adapter version.

Comment (cb-jpadilla) [reply BidMachineAdapter/build.gradle.kts:76]: Ah yes.

**Review by chauduyphanvu — APPROVED** [review]

**Pull request merged**